docs/index.rst

```diff
@@ -18,7 +18,7 @@ for Discord.
 - Sane rate limit handling that prevents 429s
 - Command extension to aid with bot creation
 - Easy to use with an object oriented design
-- Optimised for both speed and memory
+- Optimized for both speed and memory
```

Comment:
This isn't a typo. With a 'Z' is the Canadian/British spelling.

Reply:
Oh, my bad, I'll revert that then.